### PR TITLE
Add filters for time of day and day of the week

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/views/pages/dateAndTime.njk [hunk 1]
```diff
@@ -26,10 +27,86 @@
 
     {% if slotsList | length %}
     <p>Showing visit time slots for the next 28 days.</p>
+    <form action="/visit/select-date-and-time/{{ offenderNo }}" method="GET" novalidate>
+      <div class="bapv-visit-date-time-filter">
+        <h3 class="govuk-heading-s">Filter time slots</h3>
+        <div class="govuk-button-group">
+          {{ govukSelect({
```
Comment (tpmcgowan) [inline server/views/pages/dateAndTime.njk:34]: Could get the currently selected values here so they don't always just show defaults?

**Review by tpmcgowan — DISMISSED** [review]
LGTM - just comment about showing what's currently selected in the drop-downs

File: server/views/pages/dateAndTime.njk (hunk 1 above)
Comment (psoleckimoj) [reply server/views/pages/dateAndTime.njk:34]: Good point! I'll have a look.

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**